### PR TITLE
Implement OSX Travis builds

### PR DESCRIPTION
**Pull request by neoascetic**
Here the logs: https://travis-ci.org/krondor-game/minetest/builds/105706181

### CODE REVIEW

**Comment by sofar** [comment]
So travis already tested this on OSX:

https://travis-ci.org/minetest/minetest/builds/105711351

Looks good to me!

File: .travis.yml [hunk 1]
```diff
@@ -18,3 +21,7 @@ matrix:
       compiler: clang
     - env: PLATFORM=Win64
       compiler: clang
+    - env: PLATFORM=Win32
+      os: osx
+    - env: PLATFORM=Win64
+      os: osx
```
Comment (est31) [inline .travis.yml]: Can you exclude one of osx/clang or osx/gcc as well? I don't think we have to test both compilers on all platforms.

Comment (neoascetic) [inline .travis.yml]: Why not?

Comment (sofar) [inline .travis.yml]: interestingly enough the OSX clang test finished well before the Linux one...

Comment (est31) [inline .travis.yml]: last time I've checked travis said that they have only a few osx builders and therefore one should only build the neccessary. Is that now different?

**Comment by est31** [comment]
:+1: except my comment.

File: .travis.yml (hunk 1 above)
Comment (neoascetic) [inline .travis.yml]: I suppose

**Comment by est31** [comment]
:+1:

**Comment by neoascetic** [comment]
So.

**Comment by est31** [comment]
Dude I cant do more than :+1: it, bumping it all the time in github doesn't help at all.

**Comment by sfan5** [comment]
:+1:

**Comment by kilbith** [comment]
> Dude I cant do more than :+1: it

You can engage a merging-session on IRC, for example...

**Comment by est31** [comment]
c4e0d95c87a3aec22ad52ae427b704a9f3a22d21

**Pull request closed without merging**